### PR TITLE
Feature/draw on map

### PR DESCRIPTION
**Pull request by Avertere7**
### Closing [Product Backlog #16 - Design map view subpage](https://dev.azure.com/tasob/floiir/_sprints/taskboard/floiir%20Team/floiir/Sprint%201?workitem=16)

*Features:*
- Displaying map from API (API that is used by default in leaflet projects)
- Creating basic construction in frontend ( component, service, manager)
- Hiding and showing navbar with animation
- Displaying interface on map layout (search box, menu button)

### CODE REVIEW

File: frontend/web-app/src/app/components/navbar/navbar.component.html [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <header class="mat-elevation-z8">
 
-  <div class="header-container">
+  <div class="header-container" *ngIf='nav.isVisable()' [@inOutAnimation]>
```
Comment (h4570) [inline frontend/web-app/src/app/components/navbar/navbar.component.html]: Minor spelling mistake.
https://whichiscorrect.com/visible-vs-visable/

File: frontend/web-app/src/app/components/pages/map/map.component.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+export class MapComponent implements AfterViewInit {
+
+  constructor(public navbarService: NavbarService) {
+    this.mapManager = new MapManager(this.map);
+  }
+  private map;
+  mapManager: MapManager;
+  popup = L.popup();
+  searchValue = '';
+  isMenuButtonDisable = false;
```
Comment (h4570) [inline frontend/web-app/src/app/components/pages/map/map.component.ts]: Minor spelling mistake? 
Shouldnt be isMenuButtonDisabled ?

File: frontend/web-app/src/app/components/pages/map/map-manager.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+
+export class MapManager {
+
+    private map = null;
+    constructor(map) {
+        this.map = map;
+    }
+
+    public initMap(): void {
```
Comment (h4570) [inline frontend/web-app/src/app/components/pages/map/map-manager.ts:14]: Please add short description to methods/funcs

File: frontend/web-app/src/app/components/navbar/navbar.component.html [hunk 4]
```diff
@@ -60,6 +62,10 @@
 
   </div>
 
-</header>
+  <!-- <div *ngIf='!nav.isVisable()' style="text-align: center;" (mouseenter)='nav.show()'>
```
Comment (h4570) [inline frontend/web-app/src/app/components/navbar/navbar.component.html]: Please dont forget to remove commented code in future

File: frontend/web-app/src/app/components/pages/map/map-manager.ts [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+        this.map = L.map('map', {
+            center: [39.8282, -98.5795],
+            zoom: 3,
+            zoomControl: false
+        });
+        const tiles = L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
```
Comment (h4570) [inline frontend/web-app/src/app/components/pages/map/map-manager.ts]: Please consider to separate this http call to new angular service

**Review by h4570 — APPROVED** [review]

**Comment by Avertere7** [comment]
all comments were resolved, merging...

**Pull request merged**